preliz/ppls/pymc_io.py

```diff
@@ -210,12 +211,40 @@ def non_constant_parents(var_, free_rvs):
     return parents
 
 
-def parse_backfitting(model, data):
-    posterior = az.extract(data, group="posterior")
+def parse_backfitting(model, data, alternative=None):
+    """
+    Get the model information for fitting the samples from prior into user provided model's prior.
+    We need to "backfit" because we can not use arbitrary samples as priors.
 
-    model_info = get_model_information(model)[2]
-    parsed_info = [(dist, var) for var, dist in model_info.items()]
+    Parameters
+    ----------
+    model : A PyMC model
+    A probabilistic model
+
+    data : np.ndarray
```

Comment:
The input should be an inferencedata with the posterior group

Reply:
Yes changed it to `posterior` as a parameter